scripts/binary/zip.js

```diff
@@ -73,7 +73,7 @@ const checkZipSize = function (zipPath) {
   const zipSize = filesize(stats.size, { round: 0 })
 
   console.log(`zip file size ${zipSize}`)
-  const MAX_ALLOWED_SIZE_MB = os.platform() === 'win32' ? 295 : 260
+  const MAX_ALLOWED_SIZE_MB = os.platform() === 'win32' ? 295 : 300
```

Comment:
Can decrease when we remove all the old code, prior to merging this feature branch?